### DISCUSSION TITLE
【灌水】所以发帖还是用的github原本的讨论系统咯？

### DISCUSSION THREAD

**Discussion by CLENDY512** [share121/inter-knot#1596] [其他]
整个浏览界面是有了，但发帖和回复还是github本来的界面，有点割裂感。不过听说作者是高中生？那做成这样算是很棒了
![119118413_p0_master1200](https://github.com/user-attachments/assets/44189edb-79db-4a2c-838c-a3c6a3d8f351)

**Comment by huxinchi** [comment]
是的就是套壳
